### ISSUE TITLE
Play/pause from lockscreen

### ISSUE THREAD

**Issue by fedeb95** [apognu/otter#28]
with screen lock I just see the otter icon, but without buttons. Would be nice to have play, pause, next and previous too. 

Android 8.1.0

![photo_2020-02-16_14-42-22](https://user-images.githubusercontent.com/25415908/74605698-b47e1880-50ca-11ea-87c9-a6a595a7d27b.jpg)

**Comment by apognu** [comment]
Hi,

Otter already supports media controls on the lock screen and notification center. Maybe an issue specific to Android 8? I will try and reproduce this in an emulator and come back to you.

**Comment by apognu** [comment]
Aren't the controls hidden under the arrow next to the "Otter" label? What happens if you drag the card down?

**Comment by fedeb95** [comment]
Hi, thanks for the answer. If I drag the card down it asks for my unlock sequence, then if I enter it it unlocks the screen and shows me the controls in the notification

**Comment by apognu** [comment]
I could only reproduce something like this when privacy controls are enabled for Otter's notification. The setting that prevent notification contents from showing when the screen is locked (that prevent your sensitive messages to show up on your lock screen, for example) does prevent the media controls from showing up.

Could you check if Otter's notification content are blocked by this setting?

**Comment by fedeb95** [comment]
I don't have anything specifically disabled for otter, and I can see other music players controls when screen is locked. I have found some code on the web but when I tried running the app compiled from source, I had a problem that is the app continued to return to login activity. I was trying to test that line I found, right now I don't have my laptop with me so I can't be more precise. Did you ever experience something like that when developing? I'll try again and write here the exact line of code in a few days. Thanks for your support anyway. May be something wrong with my system

**Comment by apognu** [comment]
Does not ring a bell.

What was that line you found? I'm a bit at a loss, the only thing I know can hide away the media controls is this lock screen setting in Android.

**Comment by keunes** [comment]
Is lockscreen visibility [set to Public](https://developer.android.com/training/notify-user/build-notification#lockscreenNotification)? I think there was a similar issue in AntennaPod at some point. I can try & find it if you want.

**Comment by apognu** [comment]
They're not, and apparently the default is `VISIBILITY_PRIVATE`. I am wondering why I never experiences the issue myself, on any of my devices. Could this behavior be dependent on OS version or OEM specificities?

In any case, I'll implement the change, this content is not sensitive, so it couldn't hurt to be explicit.

**Comment by apognu** [comment]
This is done, it should be available in 1.0.18 that is set to be released by the end of the week (if everything goes fine).

I'll close this for now, please reopen after release if the issue is not fixed.
